CONTRIBUTING.md

```diff
@@ -2,29 +2,57 @@
 
 Operator Lifecycle Manager is Apache 2.0 licensed and accepts contributions via GitHub pull requests. This document outlines some of the conventions on commit message formatting, contact points for developers, and other resources to help get contributions into operator-lifecycle-manager.
 
-# Email and Chat
+## Communication
 
-- Email: [operator-framework][operator_framework]  
+- Slack: [#olm-dev on the Kubernetes Slack](https://kubernetes.slack.com/archives/C0181L6JYQ2)
+- Google Group: [OLM Dev Google Group](https://groups.google.com/u/1/g/operator-framework-olm-dev)
+- Mailing List: [operator-framework-olm-dev@googlegroups.com](operator-framework-olm-dev@googlegroups.com)
+- Scheduled upstream meetings can be found at the [operator-framework/community GitHub Repository][community]
+
+> Note: By joining the Google Group you will automatically be added to the mailing list.
 
 ## Getting started
 
-- Fork the repository on GitHub
+- Fork and clone the repository on GitHub.
+  -  This project uses [Go Modules](https://blog.golang.org/using-go-modules) and does not need to be cloned inside the `$GOPATH` provided that your installed version of GO is greater than or equal to 1.11.
+- Review the design documentation in the [doc/design](./doc/design) directory
```

Comment:
Will not these docs be moved to https://olm.operatorframework.io/? So, should not the `https://olm.operatorframework.io/` be inked here instead?

@anik120

Reply:
I think  `https://olm.operatorframework.io/` should be our user facing documentation, I think that should stay separate from the architecture and design docs that developers for the project should use.

Reply:
I agree with @kevinrizza on this one.